docs/tutorials/excel-power-automate-manual.md

```diff
@@ -121,4 +121,4 @@ Your flow is now ready to be run through Power Automate. You can test it using t
 
 ## Next steps
 
-Complete the [Automatically run scripts with automated Power Automate flows](excel-power-automate-trigger.md) tutorial. It teaches you how to pass data from a workflow service to your Office Script.
+Complete the [Pass data to scripts in an automatically-run Power Automate flow](excel-power-automate-trigger.md) tutorial. It teaches you how to pass data from a workflow service to your Office Script and set the Power Automate flow to run based on events instead of manual button presses.
```

Comment:
Do you need to specify manual button presses here? Maybe just leave it at "run based on events"